ls1intum/Artemis
# Added new Artemis logo

### Motivation and Context

- [x] tested on TS1


Greetings Everyone.
The public logo poll has ended and here you can see the statistics.
![image](https://user-images.githubusercontent.com/65814168/123301552-0c792a80-d51c-11eb-8bf9-e512ffee2a7f.png)

![logoRaster](https://user-images.githubusercontent.com/65814168/123301401-e6538a80-d51b-11eb-9c21-332243b88cd7.png)
This logo had the most votes.
![2a](https://user-images.githubusercontent.com/65814168/123301713-3fbbb980-d51c-11eb-94c7-b6c3553698a3.png)
After requested ajustment to TUM blue
![2A_TUMBLAU](https://user-images.githubusercontent.com/65814168/123317638-86b2aa80-d52e-11eb-9529-afd9b0803e6c.png)


This commit adds the new logo as a .png and repaces .ico file with .svg. The .svg is necessary for the smaller image in the browser tab. Imoved the favicon from public images to the root webapp folder.

The file has the following details
- .png : 232 x 201, with bit depth of 32


### Steps for Testing

Go to Artemis and look at the new Logo at the main navigation bar and at its .svg representation in your browser tab.
If you notice any errors or weird edges please tell me (I experienced a couple of issues with .ico files during prior private projects).
If the logo does not load in you tab (the svg) try looking at your static files and change the ocurrences of .ico in index.html

###Screenshots
![image](https://user-images.githubusercontent.com/65814168/123311574-a09cbf00-d527-11eb-9f74-9c27dc8d4db3.png)
![image](https://user-images.githubusercontent.com/65814168/123311669-bad69d00-d527-11eb-9597-c27097cefe89.png)
After requested ajustment to TUM blue & added transparency to favicon
![image](https://user-images.githubusercontent.com/65814168/123317560-73074400-d52e-11eb-854f-a0f854537761.png)


### Thank you 🐧👍
- Thank you all for voting during the public and internal poll (and the 2 private ones).
- Thank you @iozbeyli  , @imstreetspirit & @krusche for your constructive design advices
